ai/stt/service.py

```
@@ -1,134 +1,237 @@
+            metadata = {
+                "language": result.get("language", language),
+                "duration": result.get("duration", 0.0),
+                "segments_count": len(result.get("segments", [])),
+                "confidence": self._calculate_confidence(result.get("segments", [])),
+                "file_path": file_path
+            }
+
+            logger.info(f"Transcription completed: {transcribed_text[:50]}...")
+            return transcribed_text, metadata
+
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)